### ISSUE TITLE
Cave Default Constructor

### ISSUE THREAD

**Issue by JPGarCar** [nwplus/Factotum#208]
**Is your feature request related to a problem? Please describe.**
It is painful to create the cave every time, would be nice to have a default builder with the recommended settings. 

**Describe the solution you'd like**
The start-mentor-cave command could get a extra argument to ask for the default, or for it to be a prompt before starting the custom process.
